client/src/shared/components/SectionTitle/SectionTitle.tsx

```diff
@@ -1,10 +1,7 @@
 import style from './styles.module.scss';
+import { SectionTitleProps } from '@/types/interfaces';
 import { FC } from 'react';
 
```

Comment:
interface for props should stay here not in  src/types

it belong to this component